submission_fetcher.py

```diff
@@ -0,0 +1,71 @@
+import random as r
+import shutil
+import requests
+
+import praw
+
+reddit = praw.Reddit(client_id='AmJbMF6Sh56JNg',
+                             client_secret='1K1ovK2ku9EP_usjLR-8CFRiQB8',
+                             user_agent='windows:test:v0.1 (by u/unclesam79)')
+
+
+class SubmissionFetcher():
+
+    def __init__(self, subreddits, filter_count, filters):
+        self.subreddits = subreddits
+        self.filter_count = filter_count
+        self.filters = filters
+        self.SUBMISSIONS = []
+        self.fetch_submissions()
+
+    def fetch_submissions(self):
+        """Retrieve submissions from Reddit, add to SUBMISSIONS"""
+
+        print("Fetching submissions...")
+        self.SUBMISSIONS.clear()  # Remove previously fetched submissions
+        for sub in self.subreddits:
+            for i in range(self.filter_count):
+                for submission in reddit.subreddit(sub).top(time_filter=self.filters[i][0], limit=self.filters[i][1]):
+                    if submission not in self.SUBMISSIONS:  # Unique submissions only
+                        self.SUBMISSIONS.append(submission)
+        print(f"Got {len(self.SUBMISSIONS)} submissions in total from {self.subreddits}.")
+
+    def get_post(self):
+        """Return a random submission from SUBMISSIONS"""
+
+        if self.SUBMISSIONS == []:
```

Comment:
Better to use `if not self.SUBMISSIONS` according to [PEP-8](https://www.python.org/dev/peps/pep-0008/#id51)

Reply:
Done.